### DISCUSSION TITLE
Jumping of the page bottom in the mobile chrome on a page with a lot of swipers

### DISCUSSION THREAD

**Discussion by xfuturomax** [nolimits4web/swiper#4547] [Q&A]
Hi everyone!

I have a problem, which I described on stack:
https://stackoverflow.com/questions/67469565/jumping-of-a-sticky-footer-in-the-mobile-chrome-on-a-page-with-a-lot-of-swipers

demo: https://plnkr.co/edit/wCEUiMEmUD3f2B9u with live: https://run.plnkr.co/plunks/wCEUiMEmUD3f2B9u/

May be this is not a problem of swiper, but I tried 10k images on one page and some other heavy and complex tricks on the page, and only with swiper I see this lag/bug and only on mobile chrome.
Even with 12 sliders on a page with 3-5 pictures on each I see this issue.

Also I tried different css tricks and nothing...

Green - is bg of html tag.
![](https://media.giphy.com/media/iNQGGynwTdrmB8zRPB/giphy.gif)

**Comment by ajaypillay** [comment]
I don't have a solution but on my Pixel 3 XL running the latest version of Chrome, there's no flickering.

**Comment by jwControl** [comment]
Hi. I`m just now facing the same problem: 

https://github.com/nolimits4web/swiper/discussions/4666
https://stackoverflow.com/questions/67911858/page-jumping-during-homepage-scrolling-with-swiper-on-mobile-android

It occurs for sure on chrome and opera and NOT occurs on chrome dev and mozilla. It doesn`t depend on amount of used swipers, images or sticky footer.  Did you solve it?

**Reply by xfuturomax** [reply]
Unfortunately, I didn't. I saw a lot of sites with this side-effect, I think it's a bug of the mobile version of chrome.

**Reply by jwControl** [reply]
Thanks a lot for answer. Can you give me at least one link to website where you noticed that effect? I would like to compare it to mine.

**Reply by xfuturomax** [reply]
According to my post on stackoverflow: https://stackoverflow.com/questions/67469565/jumping-of-a-sticky-footer-in-the-mobile-chrome-on-a-page-with-a-lot-of-swipers
=> instagram:
https://giphy.com/gifs/QU8RECMfJG0OR7GO2h

airbnb.com

Actually almost the all sites with the swiper functionality
